akka-osgi/build.sbt

```diff
@@ -1,6 +1,7 @@
 import akka.{ AkkaBuild, Dependencies, Formatting, OSGi }
 
 AkkaBuild.defaultSettings
+AkkaBuild.dontPublishSettings
```

Comment:
I think that is what Patrik implied in a comment below. You think that is doable?